app/assets/javascripts/discourse/app/instance-initializers/category-badge-css-generator.js

```diff
@@ -1,33 +0,0 @@
-import { get } from "@ember/object";
```

Comment:
This has been merged with `category-color-css-generator.js`.

Reply:
This is a nice change 👍